geomstats/geometry/fiber_bundle.py

```diff
@@ -381,3 +381,48 @@ def integrability_tensor(self, tangent_vec_a, tangent_vec_b, base_point):
         https://doi.org/10.1307/mmj/1028999604.
         """
         raise NotImplementedError
+
+    def integrability_tensor_derivative(self, hor_tg_vec_x, hor_tg_vec_y,
```

Comment:
I think that tangent_vec is a bit long, but it may actually still be used. However, there needs to a clear distinction between general tangent vectors (like tangent_vec_e in this method) and horizontal tangent vectors, which are more than tangent. In this case, the method is assumed to be restricted to horizontal vectors x and y.  I propose to use horizontal_vec_x and horizontal_vec_y instead of the short hor_tg_vec that I used to stress this point. Would that fit the API?

Reply:
Yes, `horizontal_vec_x` and `horizontal_vec_y` sound great.